mkosi/backend.py

```diff
@@ -260,6 +260,17 @@ def sfdisk_spec(self) -> str:
         return ', '.join(filter(None, desc))
 
 
+def sfdisk_grain_is_supported() -> bool:
```

Comment:
Since sfdisk is already coupled to `PartitionTable` by name, this could live as a `@staticmethod` (and with a `_` in front of its name) within `PartitionTable`, since there probably won't be other users. It would also serve as a reminder once we want to remove this.

Reply:
I don't think this matters too much, sfdisk doesn't really have any tight coupling to PartitionTable so let's keep the function outside as it is.